homeassistant/components/vicare/__init__.py

```diff
@@ -42,12 +44,22 @@ class HeatingType(enum.Enum):
     fuelcell = "fuelcell"
 
 
+@contextmanager
+def catch_not_supported():
+    """Catch PyViCareNotSupportedFeatureError exceptions and return None."""
+    try:
+        yield None
```

Comment:
Why yield `None`? You can just `yield`
```suggestion
        yield
```